nemo_skills/evaluation/evaluator/segale_judge.py

```
@@ -0,0 +1,1177 @@
+            src_mt_alignments = sa.run_vecalign_explore(
+                "\n".join(src_sentences),
+                "\n".join(mt_sentences),
+                "\n".join(src_overlap),
+                "\n".join(mt_overlap),
+                src_embed,
+                mt_embed,
+                doc_id,
+                save_folder,
+                max_size=8,
+            )
+        except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)